lib/libutee/arch/arm/user_ta_entry.c

```diff
@@ -255,6 +260,11 @@ static TEE_Result entry_close_session(unsigned long session_id)
 
 	TA_CloseSessionEntryPoint(session->session_ctx);
 
+#if defined(CFG_OCALL)
+	if (tee_api_ocall_session)
```

Comment:
Could we avoid the `#if` (here and above) with the following? (untested):
```C
	if (IS_ENABLED(CFG_OCALL) && tee_api_ocall_session)
		...
```

Reply:
We could do this, but then we'd have to always declare `tee_api_ocall_session`, regardless of the value of `CFG_OCALL`. That is, if OCALL support is disabled, that variable would still exist. I would imagine that the linker would realize it's untouched and discard it.

Preference?

Reply:
> I would imagine that the linker would realize it's untouched and discard it.

Indeed, because we're using `-fdata-sections` and `--gc-sections`.
I think we should try to use `IS_ENABLED()` as much as possible, because it avoids `#ifdef` which is good for legibility and also build coverage in CI.

Reply:
Done.

Would you like to see `IS_ENABLED` for [capability reporting](https://github.com/OP-TEE/optee_os/pull/3673/files#diff-7a16795083f7db7775a0c1cf31782e9f), too?

We have there another case of a different existing style, in this case not using the macro.